astilectron.go

```diff
@@ -68,16 +74,16 @@ type Astilectron struct {
 
 // Options represents Astilectron options
 type Options struct {
-	AcceptTCPTimeout     time.Duration
-	AppName              string
-	AppIconDarwinPath    string // Darwin systems requires a specific .icns file
-	AppIconDefaultPath   string
-	BaseDirectoryPath    string
-	DataDirectoryPath    string
-	ElectronSwitches     []string
-	SingleInstance       bool
-	SkipSetup bool // If true, the user must handle provisioning and executing astilectron.
-	TCPPort              *int // The port to listen on.
+	AcceptTimeout      time.Duration
+	AppName            string
+	AppIconDarwinPath  string // Darwin systems requires a specific .icns file
+	AppIconDefaultPath string
+	BaseDirectoryPath  string
+	DataDirectoryPath  string
+	ElectronSwitches   []string
+	SingleInstance     bool
+	SkipSetup          bool // If true, the user must handle provisioning and executing astilectron.
+	TCPPort            *int // The port to listen on.
```

Comment:
Replace this attribute by `Addr string` so that users may provide their proper `addr`, not just `port`